executor.py

```diff
@@ -47,14 +48,21 @@ def __init__(
         :param overwrite_embeddings: Whether to overwrite existing embeddings. By
             default docs that have embeddings already are not processed. This value
             can be overwritten if the same parameter is passed to the request.
-        :param traversal_paths: Default traversal paths for encoding, used if
+        :param access_paths: Default traversal paths for encoding, used if
             the traversal path is not passed as a parameter with the request.
+        :param traversal_paths: please use access_paths
         :param batch_size: Default batch size for encoding, used if the
             batch size is not passed as a parameter with the request.
         """
         super().__init__(*args, **kwargs)
         self.overwrite_embeddings = overwrite_embeddings
-        self.traversal_paths = traversal_paths
+
+        if traversal_paths is not None:
+            self.access_paths = traversal_paths
+            warnings.warn("'traversal_paths' will be deprecated in the future, please use 'access_paths'")
+        else:
+            self.access_paths = access_paths
```

Comment:
how to proceed when both `traversal_paths` and `access_path` are provided?

Reply:
`traversal_paths` have higher priority